cli/lsp/semantic_tokens.rs

```diff
@@ -0,0 +1,351 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Comment:
Is this based on logic from VSCode TypeScript language service?  If so, it would be good to mention it in here at the top, so future folks can at least understand where we are getting the logic from.

Reply:
done